orga/tests/integration/components/routes/authenticated/campaign/list-test.js

```diff
@@ -43,7 +43,7 @@ module('Integration | Component | routes/authenticated/campaign/list', function(
 
       // when
       await render(hbs`<Routes::Authenticated::Campaign::List
-                  @campaigns={{campaigns}}
+                  @campaigns={{this.campaigns}}
```

Comment:
**question** : ça change quelque chose ? surtout que dans les tests rajoutés le `this` n'y est pas

Reply:
c'est rajouté, merci.